backend/prisma/schema.prisma

```diff
@@ -11,10 +11,20 @@ datasource db {
 }
 
 model User {
-  id    String    @id @default(uuid())
-  email String @unique
-  hashedPassword String
+  id                 String     @id @default(uuid())
+  email              String     @unique
+  hashedPassword     String
   hashedRefreshToken String?
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
+  createdAt          DateTime   @default(now())
+  updatedAt          DateTime   @default(now())
+  Activity           Activity[]
+}
+
+model Activity {
+  id        Int      @id @default(autoincrement())
```

Comment:
This id should be generated using uuid and type string

Reply:
I'm curious, can you tell me why it should be uuid?

Reply:
Better for scalability. The highest integer number is 2,147,483,647. If ids ever go higher than that number, it would cause problems.